dlc-manager/src/manager.rs

```diff
@@ -470,6 +474,65 @@ where
         Ok(())
     }
 
+    fn on_close_message(
```

Comment:
I'm not sure we should have that to be honest. It feels like having only the `accept` method is better. This means I can just send a close message and have it accepted by default.

Reply:
Agreed, I've updated it to simply validate the close message

Reply:
Is the idea that an application should implement a custom message handler to accept it?